datalad/interface/common_cfg.py

```diff
@@ -197,6 +200,18 @@
                'title': 'Does not execute teardown_package which cleans up temp files and directories created by tests if this flag is set'}),
         'type': EnsureBool(),
     },
+    'datalad.tests.force-fake-home': {
```

Comment:
Somehow this name is suboptimal. The whole thing seems to be more about which approach to configuration/cache isolation we are using. The new behavior is to no longer isolate the user HOME from the tests. It only "protects" the git config. User cache/config locations are in the normal HOME and leave their garbage in it.

`force-fake-home` feels like something I would want to set in order to avoid all this. But it is actually considered a fallback for outdated systems.

Reply:
Agree. I am not liking the name either

Reply:
I am failing to come up with something better. `force-outdated-isolation`? I don't know.

Reply:
re "outdated" see also https://github.com/datalad/datalad/pull/6272#discussion_r761105214